com.obsproject.Studio.Plugin.VerticalCanvas.metainfo.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.obsproject.Studio.Plugin.VerticalCanvas</id>
+  <extends>com.obsproject.Studio</extends>
+  <name>Vertical Canvas Plugin</name>
+  <developer_name>Aitum Team</developer_name>
+  <summary>Plugin for OBS Studio to add a vertical canvas by Aitum. This allows you to stream and record content in a secondary resolution to your main cavnas</summary>
```

Comment:
I'm afraid the summary needs to be a bit shorter. Summaries are usually a single sentence. Here's my uninformed, tentative suggestion: `Add a vertical canvas to stream and record in secondary resolution, by Aitum`

For context: the summary string is displayed in the add-ons list of app stores, and usually the space there is more constrained. For example:

![image](https://user-images.githubusercontent.com/3518204/216770640-a4615208-98cf-44af-9c59-76139387b663.png)